cadc-log/src/main/java/ca/nrc/cadc/log/LogControlServlet.java

```diff
@@ -167,6 +168,9 @@ public class LogControlServlet extends HttpServlet {
     private static final String LOG_CONTROL_PROPERTIES = "logControlProperties";
     static final String USER_DNS_PROPERTY = "user";
     static final String GROUP_URIS_PROPERTY = "group";
+    static final String USERNAME_PROPERTY = "username";
+    static final String SECRET_PROPERTY = "secret";
+    static final String REQUEST_HEADER_SECRET_KEY = "x-cadc-logcontrol";
```

Comment:
I was thinking only the secret param and not a custom header (just one way to do it).

Reply:
OK.